app/views/dashboard/index.html

```diff
@@ -70,13 +70,13 @@ <h2 class="govuk-heading-m">
           <p class="govuk-body">You will be emailed when {% if choices | length > 1 %}a provider{% else %}the provider{% endif %} makes a decision.</p>
 
 
-          <p class="govuk-body">Our <a href="https://beta-adviser-getintoteaching.education.gov.uk/">teacher training advisers</a> can help you prepare for any interviews.</p>
+          <p class="govuk-body">Our <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers">teacher training advisers</a> can help you prepare for any interviews. All our advisers are experienced former teachers who can give you free, one-to-one support and help you as much or as little as you need.</p>
```

Comment:
Cut this down a bit?

```suggestion
          <p class="govuk-body">Our <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers">teacher training advisers</a> can help you prepare for any interviews. Our advisers are former teachers who can give you free, one-to-one support.</p>
```